xml/dm_registers.xml

```diff
@@ -153,8 +148,8 @@
             Debug Module after power up, including the platform's system reset
             or Debug Transport reset signals.
 
-            A debugger should pulse this bit low to ensure that the Debug
-            Module is fully reset and ready to use.
+            When first connecting, a debugger should pulse this bit low to
```

Comment:
The way I see it, the DM might be in some weird state when a debugger connects, so the debugger is responsible for fixing that by pulsing reset.
If the debugger wants to do it again later, that's fine. It's allowed to, but not required.
I believe this language reflects that.